Adding styling for recent activity panel

line47 opened the pull request:
Updated fonts and added vertical pill to the left of activity and overall styling of the recent activity panel.

Commit:
Adding styling for recent activity panel

Commit:
Updated font-color on timestamp for critical errors

Commit:
Fixing lint issue

msecret commented on src/css/components/activity_log.scss:
```diff
@@ -2,82 +2,91 @@
 
-.activity_log-item_text,
+.activity_log-item_text p,
 .activity_log-item_timestamp {
```
This ruleset with:

```
activity_log-item_text p,
.activity_log-item_timestamp {
```

is already on line 30. Can the `flex-grow: 1` be moved there?

Commit:
Adjusted hierarchy to lessen specificity

Tried to remove as much nesting as possible. One changes required
changing activity log HTML so the `p` tags have the text class.

Relies on cg-dashboard#ms-activity_log_html_changes
